client/coral-framework/services/bootstrap.js

```diff
@@ -16,6 +16,25 @@ import {createStorage} from 'coral-framework/services/storage';
 import {createHistory} from 'coral-framework/services/history';
 
 /**
+ * getStaticConfiguration will return a singleton of the static configuration
+ * object provided via a JSON DOM element.
+ */
+const getStaticConfiguration = (() => {
+  let staticConfiguration = null;
```

Comment:
Usually this will not be called more than once, I wonder if it's worth using a singleton pattern here.

Reply:
The idea was that the remainder of the files here seemed like they were returned as functions, and I didn't want to introduce any other side effects by loading something that really should never change every call.